src/wizard/unity/index.md

```diff
@@ -5,16 +5,15 @@ support_level: production
 type: framework
 ---
 
+> The Unity SDK now supports line numbers for IL2CPP. The feature is currently in beta, but you can enable it at `Tools -> Sentry -> Advanced -> IL2CPP line numbers`. To learn more check out our [docs](/platforms/unity/configuration/il2cpp/).
```

Comment:
Does relative paths here work on the wizard? I think you might need https://docs.sentry.io in here